sage-ios/SAGE/SAGE/SplashView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SplashView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 4/21/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SplashView: UIView {
+    
+    var splashView = UIImageView()
+    var animated: Bool
+    let animationDuration = 1.7
+    
+    init(frame: CGRect, animated: Bool) {
+        self.animated = animated
+        super.init(frame: frame)
+        self.addSubview(self.splashView)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        self.fillWidth()
```

Comment:
call `super` here....

Reply:
i don't see you calling super here

Reply:
![Screen Shot 2020-06-18 at 4 22 25 PM](https://user-images.githubusercontent.com/4883505/85081108-e9d16a80-b17f-11ea-98ab-957969fbbd56.png)

Reply:
`super` here